### PR TITLE
fix: Rel 702 last_update form_encounter field backport

### LINKED ISSUE #7183: bug: rel_702 form_encounter missing last_update field
### Describe the bug
was added to patch.sql but need to add it to 701 to 702 upgrade too I think just for rel-702

### To Reproduce
Screenshot from 2024-01-24 12-49-53

1. Add a '....'

### Client configuration

- Browser:
- OpenEMR version:
- Operating system:

### PR DESCRIPTION
**Pull request by stephenwaite**
fixes #7183

### CODE REVIEW

**Review by bradymiller — APPROVED** [review]

File: sql/7_0_1-to-7_0_2_upgrade.sql [hunk 1]
```diff
@@ -614,3 +614,7 @@ CREATE TABLE `fee_schedule` (
     UNIQUE KEY `ins_plan_code_mod_type_date` (`insurance_company_id`, `plan`, `code`, `modifier`, `type`, `effective_date`)
 ) ENGINE=InnoDb AUTO_INCREMENT=1;
 #EndIf
+
+#IfMissingColumn form_encounter last_update
+ALTER TABLE `form_encounter` ADD `last_update` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP;
+#EndIf
```
Comment (bradymiller) [inline sql/7_0_1-to-7_0_2_upgrade.sql:620]: actually, this should not be needed since the upgrade mechanism with run patch.sql if it exists

Comment (bradymiller) [reply sql/7_0_1-to-7_0_2_upgrade.sql:620]: (so upgrade patch stuff should just need to go to database.sql and patch.sql)

**Comment by bradymiller** [comment]
was it put in database.sql in rel-702?
For the demos, is it broken in demos? :
1. Without demo data
2. With demo data
3. Both

**Comment by stephenwaite** [comment]
it's in database.sql, only broken with demo data

**Comment by bradymiller** [comment]
actually, i think i know what the issue is because i think we went through this same debugging before :)
since the patch number is not incremented, it does not run patch.sql :)
so the fix should be incrementing the patch to 1

**Comment by bradymiller** [comment]
To clarify, would increase `$v_realpatch` to 1 in version.php

**Comment by stephenwaite** [comment]
clapping

**Review by bradymiller — APPROVED** [review]

**Comment by bradymiller** [comment]
tenor99

**Pull request merged**